src/PointwiseFunctions/MathFunctions/Gaussian.hpp

```diff
@@ -85,12 +111,123 @@ class Gaussian : public MathFunction<1> {
   T apply_second_deriv(const T& x) const noexcept;
   template <typename T>
   T apply_third_deriv(const T& x) const noexcept;
+};
+
+/*!
+ *  \ingroup MathFunctionsGroup
+ *  \brief 1D Gaussian \f$f = A \exp\left(-\frac{(x-x_0)^2}{w^2}\right)\f$
+ *
+ *  \details Input file options are: Amplitude, Width, and Center. The function
+ *  takes input coordinates of type `tnsr::I<T, VolumeDim, Fr>`, where `T` is
+ * e.g. `double` or `DataVector`, `Fr` is a frame (e.g. `Frame::Inertial`), and
+ * `VolumeDim` is the dimension of the spatial volume, i.e. 2 or 3. (The case of
+ * VolumeDim == 1 is handled specially by Gaussian<1, T, Frame::Inertial>.)
+ */
+template <size_t VolumeDim, typename Fr>
+class Gaussian : public MathFunction<VolumeDim, Fr> {
+ public:
+  struct Amplitude {
+    using type = double;
+    static constexpr OptionString help = {"The amplitude."};
+  };
+
+  struct Width {
+    using type = double;
+    static constexpr OptionString help = {"The width."};
+    static type lower_bound() noexcept { return 0.; }
+  };
+
+  struct Center {
+    using type = std::array<double, VolumeDim>;
+    static constexpr OptionString help = {"The center."};
+  };
+  using options = tmpl::list<Amplitude, Width, Center>;
+
+  static constexpr OptionString help = {
+      "Applies a Gaussian function to the input coordinates"};
```

Comment:
Computes a Gaussian about an arbitrary coordinate center with given width and amplitude